flink-autoscaler/src/main/java/org/apache/flink/autoscaler/config/AutoScalerOptions.java

```diff
@@ -89,13 +89,17 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                                     + "seconds suffix, daily expression's formation is startTime-endTime, such as 9:30:30-10:50:20, when exclude from 9:30:30-10:50:20 in Monday and Thursday "
                                     + "we can express it as 9:30:30-10:50:20 && * * * ? * 2,5");
 
-    public static final ConfigOption<Double> TARGET_UTILIZATION =
-            autoScalerConfig("target.utilization")
+    public static final ConfigOption<Double> UTILIZATION_TARGET =
+            autoScalerConfig("utilization.target")
```

Comment:
Do we have to change this well-established key? This might confuse users, even though we have a fallback.

Reply:
I understand its cleaner, but perhaps we can simply add the new keys and leave this unchanged?

Reply:
> I understand its cleaner, but perhaps we can simply add the new keys and leave this unchanged?
> Do we have to change this well-established key? This might confuse users, even though we have a fallback.

I think methed `withFallbackKeys` good enough
WDYT?

Reply:
I think in the long term it's preferable to have consistent well named keys. I think the fallback keys should be enough. I would expect that users will gravitate towards the new min/max options and at that point this will make a lot of sense for everyone.